src/sentry/integrations/vercel/integration.py

```diff
@@ -142,18 +142,25 @@ def get_slug(self):
             user = client.get_user()
             return user["username"]
 
-    def get_organization_config(self):
-        vercel_client = self.get_client()
-        # TODO: add try/catch if we get API failure
-        slug = self.get_slug()
+    def _get_vercel_projects(self):
+        """If the API returns a 403, just silently replace the slug."""
```

Comment:
@mgaeta if the API returns a 403, isn't everything going to be broken related to the integration?